torchvision/models/video/resnet.py

```diff
@@ -333,6 +340,12 @@ def r2plus1d_18(pretrained=False, progress=True, **kwargs):
     Returns:
         nn.Module: R(2+1)D-18 network
     """
+    warnings.warn(
```

Comment:
The warning should appear when trying to load a state-dict, so you might need to have a custom `_load_from_state_dict` that checks the version and maybe builds a different model depending on the version? Something similar to https://github.com/pytorch/vision/pull/1224